BootloaderCorePkg/BootloaderCorePkg.dsc

```diff
@@ -374,6 +375,13 @@
   }
 !endif
 
+!if $(BUILD_CSME_UPDATE_DRIVER)
+  PayloadPkg/CsmeUpdateDriver/CsmeUpdateDriver.inf {
+    <LibraryClasses>
+      MeFwUpdateLib           | Silicon/$(SILICON_PKG_NAME)/Library/MeFwUpdateLib/MeFwUpdateLib.inf
+  }
+!endif
+

```

Comment:
I think dsc_libs in each BoardConfig.py would be better place. If we change directory name or file name, we need to touch this BootloaderCorePkg.dsc and it will impact all boards which enables BUILD_CSME_UPDATE_DRIVER.

Reply:
Moved the library to boardconfig.py